src/app/page/result/result.component.ts

```diff
@@ -98,6 +115,7 @@ export class ResultComponent implements OnInit, OnDestroy {
         error => {
           if (error instanceof HttpErrorResponse && error.status === 202) {
             console.log('Request is accepted but not handled, yet');
+            this.initPolling();
```

Comment:
Wasn't there a problem with this approach to start polling only after the request comes back? It would be ok for this PR to go back to the old behavior.

Reply:
Switched back to polling in ngOnInit().